.idea/vcs.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="IssueNavigationConfiguration">
+    <option name="links">
+      <list>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="#(\d+)" />
+          <option name="linkRegexp" value="https://github.com/arrow-kt/arrow/issues/$1" />
```

Comment:
```suggestion
          <option name="linkRegexp" value="https://github.com/arrow-kt/arrow/pull/$1" />
```
The #XXX are pull request IDs that are added automatically when merging a pull request. I'd suggest this change though I've seen that GitHub converts:

* `https://github.com/arrow-kt/arrow/issues/<ID>` into `https://github.com/arrow-kt/arrow/pull/<ID>` 
* `https://github.com/arrow-kt/arrow/pull/<ID>` into `https://github.com/arrow-kt/arrow/issues/<ID>`

according to the `<ID>`